core/db/migrate/20161012151358_opening_ledger_entries_for_store_credits.rb

```diff
@@ -0,0 +1,9 @@
+class OpeningLedgerEntriesForStoreCredits < ActiveRecord::Migration[5.0]
+  def up
+    Rake::Task["solidus:migrations:create_ledger_entries_for_store_credits:up"].invoke
```

Comment:
Why are these migrations extracted to `rake` tasks? I did this for data migrations which depended on model methods, but in this case the rake task really only has SQL and almost no ruby dependency. I believe these should be directly inside this migration.